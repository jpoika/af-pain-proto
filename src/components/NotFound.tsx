import { Link } from 'react-router';
import * as React from 'react';

const NotFound = () => {
  return (<div><h1>Page Not Found</h1>
       <p><Link to='/'>Home</Link></p>
    </div>);
};

export default NotFound;