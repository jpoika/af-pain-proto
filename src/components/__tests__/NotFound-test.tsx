import * as React from 'react';
import * as ReactDOM from 'react-dom';
import NotFound from '../NotFound';
import * as renderer from 'react-test-renderer';

test('Test NotFound Snapshot', () => {
  const component = renderer.create(
    <NotFound />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
