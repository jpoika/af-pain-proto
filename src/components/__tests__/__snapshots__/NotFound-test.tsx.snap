// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NotFound Snapshot 1`] = `
<div>
  <h1>
    Page Not Found
  </h1>
  <p>
    <a
      onClick={[Function]}
      style={Object {}}
    >
      Home
    </a>
  </p>
</div>
`;
